### DISCUSSION TITLE
Can not activate another client config in OpenVPN

### DISCUSSION THREAD

**Discussion by piperino721** [RaspAP/raspap-webgui#1715] [General]
I can upload multiple configs but only the last uploaded is active and there is no way to switch anymore.
Is this on purpose?
tnx

**Comment by billz** [comment]
This is an artifact of the upgrade to Bootstrap 5.3.3. A [fix has been pushed](https://github.com/RaspAP/raspap-webgui/pull/1716) and will be available in the next release (soon).

**Comment by piperino721** [comment]
Thank you.
